core/src/main/java/org/elasticsearch/common/MacAddressProvider.java

```diff
@@ -66,12 +60,10 @@ private static boolean isValidAddress(byte[] address) {
         try {
             address = getMacAddress();
         } catch (Throwable t) {
-            logger.warn("Unable to get mac address, will use a dummy address", t);
             // address will be set below
         }
 
         if (!isValidAddress(address)) {
-            logger.warn("Unable to get a valid mac address, will use a dummy address");
```

Comment:
Is it ok to have no feedback in this case? I guess given the initialization chain we had we weren't getting any anyway but maybe it'd be nice?